subsys/partition_manager/Kconfig

```diff
@@ -138,6 +138,18 @@ config PM_EXTERNAL_FLASH_BASE
 
 endif
 
+config BUILD_WITH_TFM
+	bool
+	select PM_SINGLE_IMAGE
+	select ENTROPY_CC3XX
+	select NORDIC_SECURITY_BACKEND
```

Comment:
@tejlmand @frkv do we need to do `select NORDIC_SECURITY_BACKEND if SOC_FAMILY_NRF`?

Reply:
Isn't partition_manager a Nordic-only. I can add the if SOC_FAMILY_NRF for safety, but my assumption is that this would only ever reach us

Reply:
Fixed

Reply:
It does reach us.
There is no safeguard, it's just grouped under partion manager menu.

Why is it actually grouped under partition manager ?
This Kconfig code has nothing to do with partition manager, from what I can tell.